README.md

```diff
@@ -7,6 +7,85 @@ Version 2.0. See the file "[LICENSE](LICENSE)" for more information.
 
 ## Introduction
 
+mod-settings is a key-value store. It is an attempt to
+make a more secure configuration system than what is
+available today in FOLIO.
+
+See [Fixing the security problem in mod-configuration](https://github.com/MikeTaylor/folio-docs/blob/main/doc/fixing-mod-configuration.md)
+for more information.
+
+It is currently implemented with PostgresQL as storage.
+
+A setting consists of these required properties
+
+* id: unique identifier (UUID)
+* scope: like namespace for the setting. This could be module.
+* key: a readable identifier; not necessarily for display
+* value: a JSON object
```

Comment:
Surely this is just a string? It happens that often that string will be a JSON document that can be deserialized into an object, but I don't think that is mod-settings's concern?

Reply:
I hate the double encoding.. But if you think that's better, ok.

Reply:
Oh, well then maybe I am misunderstanding what's going on here, and my suggested rewrite is incorrect. Let's discuss this part on Slack.